tot/datasets/__init__.py

```diff
@@ -0,0 +1,95 @@
+# make classes available upon package import
+from .dataset import Dataset  # noqa: F401
+
+"""
+Datasets
+--------
+Popular datasets. Lists all datasets that are available. For adding a new dataset, setup a new instance and
+fill respective information.
+"""
```

Comment:
I would move all of this code to a separate file and import the dataclasses here. It's best practice to not have "actual" content in the __init__ (if you can avoid it) but only manage imports